src/main/java/seedu/address/logic/parser/CommandRecorder.java

```diff
@@ -0,0 +1,23 @@
+package seedu.address.logic.parser;
+
+/**
+ * Record the latest user input for InvokeCommand.
+ */
+public class CommandRecorder {
```

Comment:
As above, perhaps "history" would represent this class better?

Reply:
I think it is good as it is now, but if you want me to change I can change.